functionary/ui/static/css/functionary.css

```diff
@@ -29,6 +29,11 @@
 .errors {
     color: red;
 }
+/* the :has() must be user enabled to work on firefox */
+/* https://developer.mozilla.org/en-US/docs/Web/CSS/:has#browser_compatibility */
+div:has(input[type="checkbox"][name="wrap"]:not(:checked)) ~ pre {
```

Comment:
This toggle doesn't seem to work in firefox for me even after enabling the feature. 

![image](https://user-images.githubusercontent.com/86254668/222568193-e565bc77-f61a-48b3-9544-b96d5e3bb8f0.png)

![image](https://user-images.githubusercontent.com/86254668/222568247-9588413c-f76b-4d24-be3c-b6d9f4c2c61c.png)

Not a show stopper, but it seemed like maybe this isn't working as intended?

Reply:
That bug you link to was opened 15 years ago, so despite some recent activity on it, I won't hold my breath for it to be fixed soon.

It doesn't really hurt anything to have it in there for now so I'm inclined to just merge this.  We can circle back and try other options that have better cross-browser compatibility at a later time.